applications/MultilevelMonteCarloLaplacianApplication/custom_python/add_custom_strategies_to_python.cpp

```diff
@@ -0,0 +1,60 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Riccardo Tosi
+//
+
+
+// System includes
+
+
+// External includes
+#include <pybind11/pybind11.h>
+
+
+// Project includes
+#include "includes/define_python.h"
+#include "custom_python/add_custom_strategies_to_python.h"
+
+#include "spaces/ublas_space.h"
+
+//strategies
+#include "solving_strategies/strategies/solving_strategy.h"
+
+//linear solvers
+#include "linear_solvers/linear_solver.h"
+
+
+namespace Kratos {
+namespace Python {
+
+void  AddCustomStrategiesToPython(pybind11::module& m)
```

Comment:
since you are not adding anything you could remove this file

Reply:
I think I can remove both "add_custom_strategies_to_python.cpp " and also "add_custom_utilities_to_python.cpp ".